pxf/pxf-hive/src/test/java/org/apache/hawq/pxf/plugins/hive/utilities/ProfileFactoryTest.java

```diff
@@ -34,31 +34,31 @@
     public void get() throws Exception {
 
         // For TextInputFormat when table has no complex types, HiveText profile should be used
-        String profileName = ProfileFactory.get(new TextInputFormat(), false);
+        String profileName = ProfileFactory.get(new TextInputFormat(), false, null);
```

Comment:
can revert back these changes now that the function with 2 arguments is back, right ?